test/fixtures/library.n/src/library/n/Button.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * ${copyright}
+ */
+
+// Provides control library.N.Button.
+sap.ui.define([
+	'./library',
+	'sap/ui/core/Control'
+], function(
+	library,
+	Control
+) {
+	"use strict";
+
+	return Control.extend("library.n.Button", {
+		metadata : {
+			library : "library.n",
+			properties : {
+				text: {type: "string", group: "Misc", defaultValue: "" }
+			},
+			aggregations: {
+				icon: {type: "sap.ui.core.Control", cardinality: "0..1" }
+			},
+			events: {
+				press: {}
+			}
+		},
+		designtime: "library/n/designtime/Button.designtime",
+		renderer: {
+			apiVersion: 2,
+			render: function(oRm, oButton) {
+				sap.ui.requireSync("./changeHandler/SplitButton");
+				oRm.openStart("button", oButton);
+				oRm.class("libNBtnBase");
+				oRm.openEnd();
```

Comment:
Why is there so much code in some of these fixtures? How is this relevant to the bundling/resources.json generation?

It's seemingly copied from somewhere. But I couldn't find it in the OpenUI5 repository. What's the origin of this code here and in the other fixtures?

Reply:
removed unnecessary code lines, I couldn't find the origin either.
I think the main purpose was to have dynamic dependencies nested in the ast